src/patternfly/components/Popover/styles.scss

```diff
@@ -0,0 +1,184 @@
+  --pf-c-popover__content--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-popover__content--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-popover__content--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-popover__content--Color: var(--pf-global--Color--100);
+  --pf-c-popover__content--BackgroundColor: var(--pf-global--BackgroundColor--100);
+
+  // Header variables
+  --pf-c-popover__header--MarginTop: var(--pf-global--spacer--xs);
+  --pf-c-popover__header--MarginBottom: var(--pf-global--spacer--sm);
+
+  // HeaderText variables
+  --pf-c-popover__header-text--PaddingTop: var(--pf-global--spacer--xxs);
+  --pf-c-popover__header-text--FontSize: var(--pf-global--FontSize--xl);
+
+  // Close
+  --pf-c-popover__close--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-popover__close--FontSize: var(--pf-global--FontSize--xl);
+
+  // Body variables
+  --pf-c-popover__body--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-popover__body--FontSize: var(--pf-global--FontSize--md);
+
+  position: relative;
+  display: inline-block;
+
+  // Elements
+  &__container {
+    // This element always needs to be light
+    --pf-c-popover__content--Color: var(--pf-global--Color--dark-100);
+    @extend %pf-t-light;
+
+    position: absolute;
+    z-index: var(--pf-c-popover__container--ZIndex);
+    display: flex;
+    width: var(--pf-c-popover__container--Width);
+    filter: var(--pf-c-popover__arrow--DropShadow);
+  }
+
+  &__arrow {
+    position: absolute;
+    border-color: var(--pf-c-popover__arrow--BorderColor);
+    border-style: solid;
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    min-width: var(--pf-c-popover__content--MinWidth);
+    padding-top: var(--pf-c-popover__content--PaddingTop);
+    padding-right: var(--pf-c-popover__content--PaddingRight);
+    padding-bottom: var(--pf-c-popover__content--PaddingBottom);
+    padding-left: var(--pf-c-popover__content--PaddingLeft);
+    font-size: var(--pf-c-popover__content--FontSize);
+    color: var(--pf-c-popover__content--Color);
+    background-color: var(--pf-c-popover__content--BackgroundColor);
+  }
+
+  &__header {
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    margin-top: var(--pf-c-popover__header--MarginTop);
+    margin-bottom: var(--pf-c-popover__header--MarginBottom);
+  }
+
+  &__header-text {
+    flex: 1;
+    padding-top: var(--pf-c-popover__header-text--PaddingTop);
+    font-size: var(--pf-c-popover__header-text--FontSize);
+  }
+
+  &__close {
+    padding-left: var(--pf-c-popover__close--PaddingLeft);
+
+    .pf-c-button.pf-m-action {
+      font-size: var(--pf-c-popover__close--FontSize);
+    }
+  }
+
+  &__body {
+    padding-right: var(--pf-c-popover__body--PaddingRight);
+    font-size: var(--pf-c-popover__body--FontSize);
+  }
+
+  // Modifiers
+  &.pf-m-left &__container {
```

Comment:
These rules should use `>`. Not that I condone a popover within a popover, but this rule could inadvertently affect other `&__container`'s nested deeper in the tree and mess up the positioning of the popover if it isn't the same as the parent.